modules/op-release-notes-1-9.adoc

```diff
@@ -77,7 +77,14 @@ In addition to the fixes and stability improvements, the following sections high
 [id="operator-new-features-1-9_{context}"]
 === Operator
 
-* With this update, you can install {pac} as a separate component, not as a part of `TektonAddon`. You can configure {pac} in the `TektonConfig` CRD.
+* With this update, {pac} is installed by default. You can disable {pac} by using the following flag:
+
++
+[source,yaml]
+
+----
+oc patch tektonconfig config --type="merge" -p '{"spec": {"platforms": {"openshift":{"pipelinesAsCode": {"enable": false}}}}}'
```

Comment:
```suggestion
$ oc patch tektonconfig config --type="merge" -p '{"spec": {"platforms": {"openshift":{"pipelinesAsCode": {"enable": false}}}}}'
```

- Also the CLI command needs to have a prompt that indicates the proper level of permissions to run it.

Reply:
Fixed.